[scroll-animations] WPT test scroll-animations/css/view-timeline-shorthand.html has incorrect contraction tests

graouts opened the pull request:
#### 3ec49d8dfd302f174cf23b2b28f2099757c16f80
<pre>
[scroll-animations] WPT test scroll-animations/css/view-timeline-shorthand.html has incorrect contraction tests
<a href="https://bugs.webkit.org/show_bug.cgi?id=265673">https://bugs.webkit.org/show_bug.cgi?id=265673</a>

Reviewed by Dean Jackson.

The last three tests of this WPT test are wrong in the same way that the equivalent `scroll-timeline` shorthand tests
were and which got fixed in 271190@main after discussion at <a href="https://github.com/web-platform-tests/wpt/issues/43336.">https://github.com/web-platform-tests/wpt/issues/43336.</a>
Indeed, while parsing, the longhands should be trimmed to match the length of `view-timeline-name`.

* LayoutTests/imported/w3c/web-platform-tests/scroll-animations/css/view-timeline-shorthand-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/scroll-animations/css/view-timeline-shorthand.html:

Canonical link: <a href="https://commits.webkit.org/271397@main">https://commits.webkit.org/271397@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/1402fe67656f260a9d19e9f8bd33313a5394205f

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/28248 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/6891 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/29606 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/30779 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/25742 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/8929 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/4271 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/26005 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/28516 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/5664 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/24316 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/4925 "Passed tests") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/42/builds/5060 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/25312 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/31468 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/25869 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/25748 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/31360 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/44/builds/5033 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/3213 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/29113 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/6614 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/25108 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/5470 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/3651 "Built successfully and passed tests") | [  ~~🛠 watch-sim~~](https://ews-build.webkit.org/#/builders/45/builds/5551 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
<!--EWS-Status-Bubble-End-->

grorg reviewed (approved):
See comment about WPT bug.

graouts committed:
[scroll-animations] WPT test scroll-animations/css/view-timeline-shorthand.html has incorrect contraction tests
https://bugs.webkit.org/show_bug.cgi?id=265673

Reviewed by Dean Jackson.

The last three tests of this WPT test are wrong in the same way that the equivalent `scroll-timeline` shorthand tests
were and which got fixed in 271190@main after discussion at https://github.com/web-platform-tests/wpt/issues/43336.
Indeed, while parsing, the longhands should be trimmed to match the length of `view-timeline-name`.

* LayoutTests/imported/w3c/web-platform-tests/scroll-animations/css/view-timeline-shorthand-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/scroll-animations/css/view-timeline-shorthand.html:

Canonical link: https://commits.webkit.org/271397@main
